modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -991,6 +993,9 @@ func (h *Handler) FinalizeUnmarshalCaddyfile(helper httpcaddyfile.Helper) error
 //	    tls_insecure_skip_verify
 //	    tls_timeout <duration>
 //	    tls_trusted_ca_certs <cert_files...>
+//		tls_trust_pool	<module> {
+//			...
+//		}
```

Comment:
Tabs and spaces in godoc are super funky. I think it needs to be like this:

```suggestion
//	    tls_trust_pool <module> {
//	        ...
//	    }
```